react/src/component/MovieComponent.js

```diff
@@ -0,0 +1,61 @@
+import React, {useEffect} from 'react';
```

Comment:
Ne bih drzao ovo unutar component/, ovo su pre pages ili screens, mozda bolje pages da bude

Reply:
mislim na MovieComponent, to je MoviePage